lib/response-transforms.js

```diff
@@ -32,4 +32,8 @@ module.exports = {
       });
     }
   },
+  jsonify: res => {
```

Comment:
What I'm saying is, if you set the accept header in the request, it's possible the server will send back the corresponding content-type header?